src/screens/ImportWalletScreen.tsx

```diff
@@ -154,15 +171,30 @@ export class ImportWalletScreen extends PureComponent<Props, State> {
       });
     } else {
       newWallet.setLabel(this.state.label || i18n.wallets.import.imported + ' ' + newWallet.typeReadable);
+      checkSubscription([newWallet], email);
```

Comment:
so how it supposed to work?  `checkSubscription`  is executed and below the user is always redirected to notifications if has an email, no matter if the wallet already is already subscribed

Reply:
that's what I was asking you, thanks

Reply:
yes, I answered you in the comment https://github.com/bitcoinvault/GoldWallet/pull/582#discussion_r560186037